app/javascript/app/components/about/about-faq/about-faq-data.js

```diff
@@ -284,9 +284,9 @@ export const sectionsData = [
       },
       {
         type: 'text',
-        title: 'Why is data missing for some countries? ',
+        title: 'Why are data missing for some countries? ',
```

Comment:
This change (`is` for `are`) is not reflected in the document

Reply:
@weberjavi Hmm, I think it is:
![Screenshot from 2019-03-15 14 11 11](https://user-images.githubusercontent.com/15097138/54437380-6202e200-472c-11e9-8e6a-4b9a05a6b4b2.png)